src/index.js

```diff
@@ -1,36 +1,103 @@
+/* eslint-disable no-use-before-define */
 import './style.css';
 import IsCompleted from './modules/Status.js';
 
-const ul = document.querySelector('ul');
-const tasks = JSON.parse(localStorage.getItem('ToDo')) || [{
-  description: 'Complete To Do list project',
-  completed: false,
-  index: 0,
-},
-{
-  description: 'Learn Object-oriented Programming',
-  completed: true,
-  index: 1,
-},
-{
-  description: 'Help Mom',
-  completed: false,
-  index: 2,
-},
-{
-  description: 'Meet friends',
-  completed: false,
-  index: 3,
-},
-];
+let tasks = JSON.parse(localStorage.getItem('ToDo')) || [];
+
+// Display Items in the UI
 
 const displayItems = () => {
```

Comment:
- [ ] As per the learning objective we should follow the ES6 syntax. Kindly try to change the function to the arrow function as per ES6 practice.

Reply:
Done 👍 .. thanks for your feedback